package.json

```
@@ -15,7 +16,13 @@
     "lint:fix": "pnpm lint -- --fix",
     "server": "pnpm --filter=@bangumi/server",
     "test": "jest",
-    "website": "pnpm --filter=@bangumi/website"
+    "website": "pnpm --filter=@bangumi/website",
+    "lint-staged": "lint-staged"
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx}": [
+      "eslint --fix"
```

```suggestion
      "pnpm lint:fix"
```
直接`eslint --fix`会不会对产出 `dist`目录也lint ?

---

dist 存在于 gitignore 规则内，这个lint-staged是对暂存区的修改内容进行校验。其它内容不会校验。因此这个顾虑不存在

---

> Run linters against staged git files and don't let 💩 slip into your code base!

lint-staged 文档原话，:D